wrapper.gradle

```diff
@@ -0,0 +1,4 @@
+wrapper {
+  gradleVersion = '6.1-rc-1'
```

Comment:
Is this mechanism something new? Is this file auto-loaded? I though gradle generated a properties file in a subfolder for this information, and that's now gitignored.

Reply:
At the time of implementing Gradle 6 was not shipped with Travis CI. I will check if it is now.